CONTRIBUTING.md

```diff
@@ -26,6 +26,7 @@ We need all contributors to sign our [volunteer agreement](https://github.com/Se
 ````
 git clone git@github.com:SexualHealthInnovations/callisto-core.git # first time only
 cd callisto-core
+brew install gnupg # first time only (make sure that gpg is installed)
```

Comment:
brew is specific to OSX, which I'd rather not have these instructions be. Let's add a note before these instructions letting people know that they need to install gpg, and link to instructions to do that--maybe [these](http://blog.ghostinthemachines.com/2015/03/01/how-to-use-gpg-command-line/)?